stable_baselines/a2c/a2c.py

```diff
@@ -192,10 +194,19 @@ def _train_step(self, obs, states, rewards, masks, actions, values, update, writ
 
         td_map = {self.train_model.obs_ph: obs, self.actions_ph: actions, self.advs_ph: advs,
                   self.rewards_ph: rewards, self.learning_rate_ph: cur_lr}
+
+        if action_masks is not None:
+            if len(action_masks) == 0:
```

Comment:
Why `len(action_mask)` could be zero? Seems like a problem coming from the environment.

Reply:
I forget what we decided: should an action mask be faked to all 1's if one isn't provided from the info dict?